.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -362,47 +362,15 @@ jobs:
           cmake --build . --config Release --parallel 4
           ./hello
 
-  spec_test_default:
-    needs: [build_iwasm, build_llvm_libraries, build_wamrc]
-    runs-on: ubuntu-20.04
-    strategy:
-      matrix:
-        test_option: [$DEFAULT_TEST_OPTIONS]
-    steps:
-      - name: checkout
-        uses: actions/checkout@v3
-
-      - name: Get LLVM libraries
-        id: cache_llvm
-        uses: actions/cache@v3
-        with:
-          path: |
-            ./core/deps/llvm/build/bin
-            ./core/deps/llvm/build/include
-            ./core/deps/llvm/build/lib
-            ./core/deps/llvm/build/libexec
-            ./core/deps/llvm/build/share
-          key: ubuntu-20.04-${{ env.LLVM_CACHE_SUFFIX }}
-
-      - name: Quit if cache miss
-        if: steps.cache_llvm.outputs.cache-hit != 'true'
-        run: echo "::error::can not get prebuilt llvm libraries" && exit 1
-
-      - name: install Ninja
-        run: sudo apt install -y ninja-build
-
-      - name: run spec tests
-        run: ./test_wamr.sh ${{ matrix.test_option }}
-        working-directory: ./tests/wamr-test-suites
-
-  spec_test_extra:
+  spec_test_all_in_one:
```

Comment:
The CI for this file isn't trigged, we should remove line 14 and line 22 in this file. it should be bugs of this file (and same for several other files). Don't remember why we add them to ignore list of pull request and push.

Reply:
I tried it on my forked repo, and although it's fine, maybe I should wait for #1566 merge, and I will rebase again.